core/domain/opportunity_services.py

```diff
@@ -381,6 +381,29 @@ def get_voiceover_opportunities(language_code, cursor):
     return opportunities, cursor, more
 
 
+def get_exploration_opportunity_summaries_by_ids(ids):
```

Comment:
Typcially, I think such functions take a list of ids and return a list of objects where the ith object corresponds to the ith id. How is this kind of representation of the return object more helpful? just curious...

Reply:
No strong opinion here, happy with what you have.